service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToStartTest.java

```diff
@@ -64,8 +67,9 @@ void mocking() {
 
     @Test
     void shouldAddConsentLabelToCaseDetails() {
-        AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToStartEvent(CaseDetails.builder()
-            .data(of("caseName", "title"))
+        AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToStartEvent(caseData().toBuilder()
```

Comment:
tbh, not big fan of this approach, it suggest that certain things happens for RETURNED state, but it is just technical trick to not trigger certain code. I think it would be better to
add private  method 
```
    private static FeesData feesData(long amount) {
        return FeesData.builder()
            .totalAmount(BigDecimal.valueOf(amount))
            .build();
    }
```

then do:
```
 @Test
    void shouldAddConsentLabelToCaseDetails() {
        given(feeService.getFeesDataForOrders(any())).willReturn(feesData(10));

        AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToStartEvent(caseData());

        assertThat(callbackResponse.getData())
            .containsEntry("submissionConsentLabel",
                "I, Emma Taylor, believe that the facts stated in this application are true.");
    }
```

you can use new method in other places in this class